backend_py/primary/primary/routers/explore/router.py

```diff
@@ -6,13 +6,15 @@
 from primary.services.sumo_access.case_inspector import CaseInspector
 from primary.services.sumo_access.sumo_inspector import SumoInspector
 from primary.services.utils.authenticated_user import AuthenticatedUser
+from primary.middleware.add_browser_cache import no_cache
 
 from . import schemas
 
 router = APIRouter()
 
 
 @router.get("/fields")
+@no_cache
```

Comment:
This probably could be cached. It very rarely changes - and when it does the user has to wait for access to sync through infrastructure and Azure anyway. E.g. `max-age` of one minute and a default `stale-while-revalidate` of one day?